onnx/proto_utils.h

```diff
@@ -6,12 +6,12 @@
 namespace onnx {
 
 template <typename Proto>
-bool ParseProtoFromBytes(Proto* proto, const char* buffer, size_t length) {
+bool ParseProtoFromBytes(Proto& proto, const char* buffer, size_t length) {
```

Comment:
btw, Google's style guide recommends to use pointers isntead of non-const reference for output params. Do we want to follow this style?